app/controlplane/pkg/data/workflowrun.go

```diff
@@ -236,7 +236,7 @@ func entWrToBizWr(wr *ent.WorkflowRun) *biz.WorkflowRun {
 	}
 
 	if wf := wr.Edges.Workflow; wf != nil {
-		w, _ := entWFToBizWF(wf, nil)
+		w, _ := entWFToBizWF(context.TODO(), wf, nil)
```

Comment:
TODO?

Reply:
yeah, we do not have the context there and I thought it might not be worth updating all the code, but let me double check

Reply:
updated, thanks for pointing it out